modules/dashboard/job/variables.tf

```diff
@@ -7,3 +7,8 @@ variable "labels" {
   description = "Additional labels to apply to the dashboard."
   default     = {}
 }
+
+variable "project_id" {
+  description = "ID of the GCP project"
+  type        = string
+}
```

Comment:
While we're in here, can we plumb through `notification_channels` to any relevant sections (e.g. `resources`)?

I bring it up because this is going to be a breaking change downstream, and I'd love to knock off as many things as possible in one go.

Reply:
done.